src/main/java/dev/waterdog/waterdogpe/ProxyServer.java

```diff
@@ -428,14 +428,7 @@ public ServerInfo getServerInfo(String address, int port) {
      */
     public ServerInfo getForcedHost(String serverHostname) {
         Preconditions.checkNotNull(serverHostname, "ServerHostname can not be null!");
-        String serverName = null;
```

Comment:
Same here

Reply:
Fixed in [8a058ff](https://github.com/WaterdogPE/WaterdogPE/pull/216/commits/8a058ff55bab2328119d76d8b59761bb9b30c853)